src/Servers/Kestrel/test/InMemory.FunctionalTests/Http2/Http2ConnectionTests.cs

```diff
@@ -1936,6 +1936,58 @@ public async Task DATA_Sent_DespiteStreamOutputFlowControl_IfEmptyAndEndsStream(
         await StopConnectionAsync(expectedLastStreamId: 1, ignoreNonGoAwayFrames: false);
     }
 
+    [Fact]
+    public async Task StreamWindow_BiggerThan_ConnectionWindow()
+    {
+        // This only affects the stream windows. The connection-level window is always initialized at 64KiB.
+        _clientSettings.InitialWindowSize = 16384 * 5;
+
+        await InitializeConnectionAsync(async context =>
+        {
+            var data = new byte[16384 * 4];
+            data.AsSpan().Fill(1);
```

Comment:
Why are we filling the array with 1s? Doesn't `new byte[16384 * 4]` fit just inside the 64KiB connection window? Apparently it's one byte too large given the test, but I'm trying to understand why.

Reply:
https://github.com/dotnet/aspnetcore/blob/46554c9183e7ebdda173edd03038f19ea7ac75d3/src/Shared/ServerInfrastructure/Http2/Http2PeerSettings.cs#L14

Reply:
Right, it's *almost* 64KiB.